var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -123,6 +123,8 @@ class Vtk(CMakePackage):
     patch("vtk_find_liblzma.patch", when="@8.2")
     patch("vtk_movie_link_ogg.patch", when="@8.2")
     patch("vtk_use_sqlite_name_vtk_expects.patch", when="@8.2")
+    patch("vtk_proj_include_no_strict.patch", when="@9: platform=windows")
+    patch("vtk_alias_hdf5.patch", when="@9: platform=windows")
```

Comment:
(minor) Generally for each patch it's good to either (a) explain what it's for or (b) link to whatever upstream discussion relates to it.

(b) is always preferred if available. If not, an explanation would be helpful if anyone needs to extend the patch later.